config/redis.conf

```diff
@@ -450,7 +450,7 @@ slave-priority 100
 # limit for maxmemory so that there is some free RAM on the system for slave
 # output buffers (but this is not needed if the policy is 'noeviction').
 #
-# maxmemory <bytes>
+maxmemory 64mb
```

Comment:
Is this the recommended amount via Pathfinder?

Reply:
Yes, but it does not seem to take, at least PF does not see it, Redis might need a config file reload before starting

Reply:
Might need to add it to the entrypoint script